DOC-2267: Add new documentation for `ImportWord` Plugin. (#3094)
tinymce/tinymce-docs: modules/ROOT/partials/commands/importword-cmds.adoc

[cols="1,3",options="header"]
|===
|Command |Description

> |ImportWord |Imports `.docx` or `.dotx` files into the editor.

Author: ```suggestion
|ImportWord |Opens a file picker, that allows content from either a `.docx` or `.dotx` file to be imported into the editor.
```

|===

.Example
[source,js]
----
tinymce.activeEditor.execCommand('ImportWord');

> ----

Reviewer: Add, please, the Note that the ImportWord command should be initiated through a user activation, such as a button click.

Author: ```suggestion
----

[NOTE]
To activate the `ImportWord` command, users are required to click on the "Import to Word" button. Upon clicking the button, the `ImportWord` command is triggered.
```

Reviewer: **That is not pretty what I intended.**
Unlike other commands, this one can not be directly invoked via a script at the top level or from the browser console. Instead, it requires a specific user action within the user interface (UI), such as clicking a toolbar button (it may be any custom button, not necessarily the 'ImportWord' registered button). To ensure proper execution, the command should be called within the scope of an event handler of specific user interaction.

[cols="1,3",options="header"]
|===
|Command |Description
|ImportWord |Opens a file picker, that allows content from either a `.docx` or `.dotx` file to be imported into the editor.
|===

.Example
[source,js]
----

> tinymce.activeEditor.execCommand('ImportWord');

Reviewer: ```suggestion
document.querySelector('button').addEventListener('click',
  () => tinymce.activeEditor.execCommand('ImportWord')
);
```

----

[NOTE]

> To activate the `ImportWord` command, users are required to click on the "Import to Word" button. Upon clicking the button, the `ImportWord` command is triggered.

Reviewer: ```suggestion

The `ImportWord` command cannot be directly invoked through a script at the top level or from the browser console. Instead, it necessitates a specific user action within the user interface, such as clicking a toolbar button. To ensure correct execution, the command should be called within the scope of an event handler tied to a specific user interaction.
```

[cols="1,3",options="header"]
|===
|Command |Description
|ImportWord |Opens a file picker, that allows content from either a `.docx` or `.dotx` file to be imported into the editor.
|===

.Example
[source,js]
----
document.querySelector('button').addEventListener('click',
  () => tinymce.activeEditor.execCommand('ImportWord')
);
----

> [NOTE]
> 

Author: ```suggestion
[NOTE]
```

> [NOTE]
> 
> The `ImportWord` command cannot be directly invoked through a script at the top level or from the browser console. Instead, it necessitates a specific user action within the user interface, such as clicking a toolbar button. To ensure correct execution, the command should be called within the scope of an event handler tied to a specific user interaction, as shown in the example.

Author: ```suggestion
[NOTE]
The `ImportWord` command cannot be directly invoked through a script at the top level or from the browser console. Instead, it necessitates a specific user action within the user interface, such as clicking a toolbar button. To ensure correct execution, the command should be called within the scope of an event handler tied to a specific user interaction, as shown in the example.
```
